### PR TITLE
21293 list journeys

### PR DESCRIPTION
**Pull request by nnorbert**
List existing journeys and journey details page.

### CODE REVIEW

File: apps/integrator/libraries/profile-authentication-factory.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      ),
+      { headers: { cookie: cookieHeader } },
+    );
+    const { token } = await res.json();
+    return token;
+  }
+
+  static async getProfileClient(organizationId: string): Promise<Profile> {
+    const token = await this.getToken(organizationId);
+    console.log(">>>", token);
```
Comment (peschina) [inline apps/integrator/libraries/profile-authentication-factory.ts:31]: Can we remove this log?

File: apps/integrator/messages/en.json [hunk 2]
```diff
@@ -43,6 +43,29 @@
         "titleLabel": "Title",
         "urlLabel": "Message template URL"
       }
+    },
+    "existingJourneys": {
+      "title": "Existing journeys",
+      "empty": {
+        "title": "There are no journes yet!",
```
Comment (peschina) [inline apps/integrator/messages/en.json:50]:
```suggestion
        "title": "There are no journeys yet!",
```

**Review by peschina — COMMENTED** [review]
Looks good, a couple of comments

File: apps/integrator/libraries/profile-authentication-factory.ts (hunk 1 above)
Comment (nnorbert) [reply apps/integrator/libraries/profile-authentication-factory.ts]: Done!

File: apps/integrator/app/[locale]/(hosted)/admin/journeys/details/[journeyId]/page.tsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    return notFound();
+  }
+
+  return result.rows[0];
+};
+
+const getUserName = async (userId: string, profileApi: Profile) => {
+  const user = await profileApi.getUser(userId);
+  return `${user.data?.firstName} ${user.data?.lastName}`;
+};
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply apps/integrator/app/[locale]/(hosted)/admin/journeys/details/[journeyId]/page.tsx:49]: Done!

**Pull request merged**